### PR TITLE
"load more" for wiki contributions in user page

### PR DESCRIPTION
**Pull request by kerspoon**
This is a work in progress that might be useful for others to see. 

It sometimes doesn't get anything in the `results[n].tag` and sometimes does, needs filtering at the DB layer.

### CODE REVIEW

**Comment by kerspoon** [comment]
(Adding a couple of screenshots before I delete them locally )

Sometime tag data loads properly
taginfo

Might have something to do with documentId being null on some DB items, this is surely an error but might not be the same error.
image

File: packages/lesswrong/lib/collections/revisions/views.ts [hunk 1]
```diff
@@ -8,9 +8,21 @@ declare global {
...
     after?: Date|string|null,
+    userId?: string
   }
 }
 
+Revisions.addView('revisionsByUser', (terms: RevisionsViewTerms) => {
+  return {
+    selector: {
+      userId: terms.userId,
+      // tag: {$ne:null} for some reason this returns a view where tag is often null. if we filter by this then it returns nothing.
```
Comment (jpaddison3) [inline packages/lesswrong/lib/collections/revisions/views.ts]: The thing that's happening here is that `tag` is a resolver-only field. Those fields only exist when you ask our API for them, and are not stored in the database.

https://github.com/centre-for-effective-altruism/EAForum/blob/9f2d16e1767bc7d24d3e12d5d32bebb32a310a5d/packages/lesswrong/lib/collections/revisions/schema.ts

Comment (jpaddison3) [reply packages/lesswrong/lib/collections/revisions/views.ts]: I changed the query to specify `{collectionName: 'Tags'}` and everything worked.

**Review by jpaddison3 — APPROVED** [review]
LGTM

**Pull request merged**